docs/request-options.rst

```diff
@@ -975,6 +975,27 @@ timeout
     // PHP Fatal error:  Uncaught exception 'GuzzleHttp\Exception\RequestException'
 
 
+.. force_ipv4-option:
+
+force_ipv4
+---------------
+
+:Summary: Set to true if you want the HTTP handlers to use only ipv4 protocol.
+:Types: bool
+:Default: false
+:Constant: ``GuzzleHttp\RequestOptions::FORCE_IPV4``
+
+.. code-block:: php
+
+    // Force ipv4 protocol
+    $client->request('GET', '/foo', [GuzzleHttp\RequestOptions::FORCE_IPV4 => TRUE]);
```

Comment:
1, Done.
2, Done, but i strongly recommend to use the constant, not the string.